src/mixins/widget.js

```diff
@@ -72,6 +67,11 @@ Read more on using connectors: https://alg.li/vue-custom`
     },
   },
   methods: {
+    getParentIndex() {
```

Comment:
not sure if it's the best approach, though

Reply:
this makes sense, but would need to see it together with the InstantSearch change

Reply:
@Haroenv can you explain what you mean here? ↑